src/commands/lambda/instrument.ts

```diff
@@ -4,7 +4,7 @@ import {Command} from 'clipanion'
 import {parseConfigFile} from '../../helpers/utils'
```

Comment:
What's supposed to happen if uninstrumentation fails on one lambda but succeeds on the others?

Reply:
I think it will tell the user that everything failed, we might have to consider catching the error and stating that it failed only in certain ones, will take a look into it, but won't do it in this PR.

Reply:
That sounds great!